Packs/ctf01/ReleaseNotes/1_0_31.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Cortex XDR - IR CTF
+
+- Updated the error message on alert_id input when core-get-cloud-original-alerts fails when debugging playbooks.
```

Comment:
Updated the error message on the `alert_id` input when the `core-get-cloud-original-alerts` command fails while using the playbook debugger.